tests/test_freshness_date_parser.py

```diff
@@ -576,6 +593,19 @@ def test_relative_past_dates(self, date_string, ago, period):
 
     @parameterized.expand([
         # English dates
+        param('1 fortnight', ago={'days': 14}, period='day'),
+        param('last fortnight', ago={'days': 14}, period='day'),
+        param('14 fortnight', ago={'days': 196}, period='day'),
+        param('a fortnight ago', ago={'days': 14}, period='day'),
+        param('last fortnight', ago={'days': 14}, period='day'),
```

Comment:
Same as above. It is repeated as well.

Reply:
@gutsytechster Done these changes, Please Look into it 😊.